SPECS-EXTENDED/lensfun/lensfun.spec

```diff
@@ -0,0 +1,383 @@
+%if !0%{?bootstrap} && (0%{?fedora} || 0%{?rhel} > 6)
+%global tests 1
+%global python3 python%{python3_pkgversion}
+%endif
+Summary:        Library to rectify defects introduced by photographic lenses
+Name:           lensfun
+Version:        0.3.2
+Release:        26%{?dist}
+License:        LGPLv3 AND CC-BY-SA
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://lensfun.sourceforge.net/
+Source0:        http://downloads.sourceforge.net/%{name}/%{name}-%{version}.tar.gz
+## upstream patches
+Patch1:         0001-Only-require-glib-2.40-when-tests-are-build-without-.patch
+Patch38:        0038-Only-use-proper-C-new-and-delete-syntax-for-object-c.patch
+Patch58:        0058-Use-database-in-source-directory-while-running-tests.patch
+Patch59:        0059-Patch-47-respect-DESTDIR-when-installing-python-stuf.patch
+Patch60:        0060-Various-CMake-patches-from-the-mailing-list.patch
+Patch113:       0113-Added-std-namespace-to-isnan.patch
+## upstreamable patches
+# install manpages only when INSTALL_HELPER_SCRIPTS=ON
+Patch200:       lensfun-0.3.2-INSTALL_HELPER_SCRIPTS.patch
+## upstream patches (master branch)
+Patch866:       0866-Pull-isnan-into-std-namespace-include-cmath-not-math.patch
+# for rst2man, if INSTALL_HELPER_SCRIPTS != OFF
+BuildRequires:  %{_bindir}/rst2man
+BuildRequires:  cmake >= 2.8
+BuildRequires:  doxygen
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(glib-2.0)
+BuildRequires:  pkgconfig(libpng)
+BuildRequires:  pkgconfig(zlib)
+%if 0%{?python3:1}
```

Comment:
We always have Python 3 available - let's remove this check along with the `%else` part.